### DISCUSSION TITLE
CSS Q&A

### DISCUSSION THREAD

**Discussion by odinharindra** [odinharindra/training#3] [Q&A] [answered]
This thread is for CSS Q&A discussions

**Comment by Rakhattri** [comment]
Sir, to ask you about CSS i would like to know the trends on the industry or the best practice in the industry concerning using CSS The Box model or Flex-box model which is being developers go to choice and why ?

**Reply by odinharindra** [reply, marked as answer]
We can define the CSS layout by using 
1. Float property
2. Position property
3. Flex-box layout, and
4. Grid layout

Depending on the requirement we choose appropriate way.
Grid system is most used for responsive layout and flex-box is used for arranging child elements as desired.
float is very rarely used where as Position property used occasionally
